README.md

```diff
@@ -23,28 +23,28 @@ Additionally, please make sure that you're using the version of Lean that the cu
   curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh
   ```
   If this also fails, follow the instructions under `Regular install` [here](https://leanprover-community.github.io/get_started.html).
-* To build `std4` run `lake build`. To build and run all tests, run `make`.
-* If you added a new file, run the command `scripts/updateStd.sh` to update the
+* To build `batteries` run `lake build`. To build and run all tests, run `make`.
+* If you added a new file, run the command `scripts/updateBatteries.sh` to update the
   imports.
 
 # Documentation
 
-You can generate `std4`'s documentation with
+You can generate `batteries`'s documentation with
```

Comment:
Should this be "`batteries`'" instead? (This might also apply in other places.)

Reply:
Thanks, looks like the only one.